### PR TITLE
Added bcrypt hash script.

### PR DESCRIPTION
**Pull request by ashcrow**
Adds a new command, ``commissaire-hashpass`` which makes it easy to create bcrypt hashed passwords for use.

### CODE REVIEW

**Comment by mbarnes** [comment]
Don't know if this is a bug or missing dependency or what, but trying the command after `./setup.py install` I get this:
```
(commissaire)mbarnes@mbarnes:~/git/github/users/ashcrow/commissaire$ commissaire-hashpass 
Traceback (most recent call last):
  File "/home/mbarnes/commissaire/bin/commissaire-hashpass", line 9, in <module>
    load_entry_point('commissaire==0.0.0', 'console_scripts', 'commissaire-hashpass')()
  File "/home/mbarnes/commissaire/lib/python2.7/site-packages/pkg_resources/__init__.py", line 519, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/home/mbarnes/commissaire/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2630, in load_entry_point
    return ep.load()
  File "/home/mbarnes/commissaire/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2310, in load
    return self.resolve()
  File "/home/mbarnes/commissaire/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2316, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
ImportError: No module named py
```

File: setup.py [hunk 1]
```diff
@@ -48,6 +48,7 @@ def extract_requirements(filename):
     entry_points={
         'console_scripts': [
             'commissaire = commissaire.script:main',
+            'commissaire-hashpass = commissaire.hash_pass_script.py:main',
```
Comment (mbarnes) [inline setup.py]: Lose the `.py` here, right?  It was causing the error I initially reported.

Comment (ashcrow) [inline setup.py]: :dizzy_face: :palm_tree: 

Will do!

File: src/commissaire/hash_pass_script.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    """
+    Generates a bcrypt hash for a given password.
+    """
+    parser = argparse.ArgumentParser(
+        description='bcrypt based password hasher.',
+    )
+    parser.add_argument('-r', '--rounds', type=int, default=12, help='')
+    parser.add_argument(
+        '--stdin', action='store_true', default=False,
+        help='Reads password from stdin instead of prompting.')
```
Comment (mbarnes) [inline src/commissaire/hash_pass_script.py]: Could make this a tad more flexible with a readable `argparse.FileType` argument , which understands `'-'` to mean `sys.stdin` (and rename the option to something like `--pwfile`).

If you think it's worth it... might be overkill.

**Comment by mbarnes** [comment]
GTG as is.  I'll let you decide on the `argparse` suggestion.  :+1:

File: src/commissaire/hash_pass_script.py (hunk 2 above)
Comment (ashcrow) [inline src/commissaire/hash_pass_script.py]: I like it! I'll make the change shortly.

Comment (ashcrow) [inline src/commissaire/hash_pass_script.py]: Done.

**Comment by mbarnes** [comment]
Nice!  Will merge shortly.

**Comment by mbarnes** [comment]
Squashed and merged.

**Pull request closed without merging**